README.md

```diff
@@ -309,12 +321,36 @@ Add the version of `solc` you want to use into `index.html`:
 <script
   type="text/javascript"
   src="https://binaries.soliditylang.org/bin/{{ SOLC VERSION }}.js"
+  integrity="sha256-{{ BASE64-ENCODED HASH OF SOLC VERSION  }}"
+  crossorigin="anonymous"
 ></script>
 ```
 
-(Alternatively use `https://binaries.soliditylang.org/bin/soljson-latest.js` to get the latests version.)
+This will load `solc` into the global variable `window.Module`.
```

Comment:
Into the global variable or is that a namespace / object? Maybe saying assigning it as a member of `window.Module` would be more precise?

Reply:
I think `global variable` is correct. I mean, the `Module` is a property of window and thus a global variable since window is a global object of a web page that refers to itself. I may be wrong, but solc is not assigned as a member of the `window.Module`, it is loaded as the Module object in the window. Maybe `global object` would be correct, too. Like:
```
This will load solc as the global object window.Module.
```

But honestly, I'm not sure about the terminology. I usually refer to any window property as global variables, but I may be calling them by the wrong name, haha